.github/workflows/test_branches.yml

```diff
@@ -92,7 +92,7 @@ jobs:
           skip_doctest: 1
           TARGET: linux
           PYENV: conda
-          PACKAGES: mpi4py
+          PACKAGES: mpi4py==3.1.5
```

Comment:
What happens if you do the following instead?

```yaml
 PACKAGES: "mpich mpi4py"
```

Reply:
That causes different failures but it works for `openmpi mpi4py`! I have a new PR open for it (#3239 ). Thanks for helping us work this out!

Reply:
You should submit this as an issue to https://github.com/conda-forge/intel_repack-feedstock. For some unknown reason `conda install impi_rt` is failing, and that's quite weird.